docs/diagrams/sal-search.plantuml

```diff
@@ -0,0 +1,58 @@
+[billedeurl|
```

Comment:
This should not be a part of the plantuml?

Reply:
you wrote it! :D - will fix.... :)